quickstart/expose-keptn.sh

```diff
@@ -121,13 +121,20 @@ do
   # echo "retries:  $retries / $MAX_RETRIES" 
   if [ ${http_code} -eq 200 ]; then
     echo "Attempting to open Keptn bridge on http://$INGRESS_IP.nip.io:$INGRESS_PORT/bridge"
-    if ! command -v open &> /dev/null
+
+     if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+      OPEN=('xdg-open')
+    else
+        OPEN=('open')
```

Comment:
Indentation is a little off here :)

Reply:
That is odd because my editor displays them the same. I will automatically format the document, which should work.